packages/contract/lib/execute.js

```diff
@@ -87,7 +88,9 @@ const execute = {
       args = processedValues.args;
       params = processedValues.params;
     }
-
+    if (isCall) {
```

Comment:
Sorry, one more nitpick; would you mind leaving a comment here about not needing to make a call for the network info in the case of making a "call"? Add that and then I am good to go on this! Thanks for doing this work @msoni89!

Reply:
Alternative suggestion: Perhaps instead of `isCall`, the variable could be called something like `useCachedNetworkId`?  In that case, the comment would not go here, but rather where `true` is passed for it.  (Either way works I think.)  But yes, thanks for doing this!